package.nls.json

```diff
@@ -141,7 +141,7 @@
     "AWS.command.login": "Connect to AWS",
     "AWS.command.login.cn": "Connect to Amazon",
     "AWS.command.logout": "Sign out",
-    "AWS.command.createIssueOnGitHub": "Create a New Issue on GitHub",
+    "AWS.command.createIssueOnGitHub": "Report Toolkit Issue",
```

Comment:
"Create a New Issue on GitHub" could be confused with general GitHub functionality, as opposed to a Toolkit-specific action.